data/remnant/remnant 2 cognizance.txt

```diff
@@ -1149,7 +1149,7 @@ mission "Remnant: Cognizance 25"
 	on offer
 		event "remnant: ssil vida inhabited"
 		conversation
-			`You have barely had a chance to check to finish your pre-flight checks on <ship> before Gavriil is tapping on your airlock. "Setting up a new environment is always an interesting challenge," Gavriil gestures. "But to the task at hand: Could you do a quick patrol through this collection of newly accessible systems? We have scouts en route with surveillance satellites to place in each of them, but I picked up the signature of a Korath Raider that was here recently, and I want to be sure that it left no surprises." They pause, then make a gesture you have come to recognize as meaning the accompanying message is intended as a summary. "So, visit each of the systems in this sector, and make sure that the raider did not leave anything behind." Their gestures switch to a tone of finality. "Can you help with this?"`
+			`You have barely had a chance to check to finish your pre-flight checks on <ship> before Gavriil is tapping on your airlock. "Setting up a new environment is always an interesting challenge," Gavriil gestures. "But to the task at hand: Could you do a quick patrol through this collection of newly accessible systems? We have scouts en route with surveillance satellites to place in each of them, but I picked up the signature of a Palavret that was here recently, and I want to be sure that it left no surprises." They pause, then make a gesture you have come to recognize as meaning the accompanying message is intended as a summary. "So, visit each of the systems in this sector, and make sure that the Palavret did not leave anything behind." Their gestures switch to a tone of finality. "Can you help with this?"`
```

Comment:
Not suggesting this should change, just for future reference:
Is it *not* appropriate to describe the Palavret as a "raider" (noun, not proper noun)?
Is raiding not still what it usually does?

Though, this is the ship that left behind the 3 'olofezs?
That's supposed to be the Tublafet (Raider 42)?
With the old names, I think it's perfectly reasonable for characters to call it a "Korath Raider", but now that th names are actually different, unless they just couldn't tell that it was different and actually thought it was the same model, it might make sense to use a generic term, like "raider".

Reply:
In general, when going through mission files, when someone was speaking about an uncertain ship, or korath ships in general, I used small r "raider."  When speaking *specifically* about a ship that is known (as far as I can tell) to be a Palavret, then I use "Palavret."  Big R raider is no longer appropriate, I think.

In this case, if this is where in the story I think it is, I think it might be appropriate to say Palavret, because they haven't seen a Tubfalet, a Palavret is very similar, and the signature may obviously not be that of a Rano'erek, they may mistakenly call it a Palavret.